eng/tools/eng-package-utils/index.js

```diff
@@ -33,46 +33,79 @@ async function writePackageJson(filename, contentObject) {
   }
 }
 
-//This gets the list of rush packages as well as their packageJsons
-//This is specifically used in set-dev-dependencies script
-const getRushPackageJsons = async repoRoot => {
-  const rushPath = path.resolve(path.join(repoRoot, "rush.json"));
-  const baseDir = path.dirname(rushPath);
-  const rushJson = parse(await readFile(rushPath, "utf8"));
+function getVersionPolicyName(project) {
+  const packageName = project.manifest.name;
+  const packageDir = project.rootDirRealPath;
+  if (packageName.startsWith("@azure-tests/")) {
+    return "test";
+  } else if (
+    packageName.startsWith("@azure-tools/") ||
+    packageName === "@azure/dev-tool" ||
+    packageName === "@azure/eslint-plugin" ||
+    packageName === "@azure/mock-hub" ||
+    packageName === "@azure/storage-internal-avro"
+  ) {
+    return "utility";
+  } else if (packageName.includes("/arm-")) {
+    return "management";
+  } else if (packageDir.includes("sdk/core") || packageDir.includes("sdk\\core")) {
+    return "core";
```

Comment:
logger, abort-controller, and unbranded are not handled by this case, are they?

Reply:
The condition is checking the directory name sdk/core/, they should be under this directory.